src/main/java/mate/academy/mybookshop/repository/BookRepositoryImpl.java

```diff
@@ -38,6 +39,15 @@ public BookEntity save(BookEntity book) {
         }
     }
 
+    @Override
+    public BookEntity getById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.get(BookEntity.class, id);
+        } catch (Exception e) {
+            throw new EntityNotFoundException("Can't get book by id: " + id, e);
```

Comment:
EntityNotFoundException is meant for service use only.

Reply:
sorry, i don't get it, why? ) 
for me is one place to try catch  and throw this exception for now
maybe, if that was `findById` method, i will do it with Optional and made `orElseThrow` in Service class 
Please help me out with my opinion. )